.gitignore

```diff
@@ -3,6 +3,7 @@
 *.pyc
 config.py
 forums/wsgi.py
+forums/local.py
```

Comment:
You should never push usernames and passwords to the GIT.
Please replace the values with dummy data.